skrifa/src/outline/autohint/hint.rs

```diff
@@ -366,6 +366,34 @@ mod tests {
         assert_eq!(coords, expected_coords);
     }
 
+    // Specific test case for <https://issues.skia.org/issues/344529168>
+    #[test]
+    fn skia_ahem_test_case() {
+        let font = FontRef::new(font_test_data::AHEM).unwrap();
+        let outline = hint_latin_outline(
+            &font,
+            24.0,
+            Default::default(),
+            5,
```

Comment:
Nit: If `hint_latin_outline` would take a `GlyphId` parameter instead of a u32, this magic number parameter `5` here would perhaps be a bit easier to read/understand? Optional: Also, perhaps add a comment that this is the typical Ahem block square?

Reply:
Good suggestions. Done and done.